xs2a-adapter-service-api-adapter/src/main/java/de/adorsys/xs2a/adapter/http/StringUri.java

```diff
@@ -84,4 +84,8 @@ private static boolean startsWithVersion(String stringToCheck) {
     public static boolean containsProtocol(String urlToCheck) {
         return urlToCheck.contains("://");
     }
+
+    public static boolean isNotBlank(String requestBody) {
```

Comment:
As far as I see, `StringUri` class is used to provide a bunch of services for URI and URL Strings. I am not sure, that this method should be located in `StringUri` class as well, as it is not related to URI and URL anyhow.
What do you think?

Reply:
thanks, removed